apprclient.go

```diff
@@ -176,6 +178,28 @@ func (c *Client) PromoteChart(name, release, channel string) error {
 	return nil
 }
 
+// DeleteRelease removes a release from the server.
+func (c *Client) DeleteRelease(name, release string) error {
```

Comment:
Would be cool to order the methods alphabetically.

Reply:
Sure, done.